pkgs/development/python-modules/mizani/default.nix

```diff
@@ -12,16 +12,16 @@
 
 buildPythonPackage rec {
   pname = "mizani";
-  version = "0.10.0";
-  format = "pyproject";
+  version = "0.9.3";
```

Comment:
why is a downgrade necessary here?

Reply:
In the end it was not.
I was testing things around.

Reply:
Actually, plotnine==0.12.4 is not compatible with mizani==0.10.0.
As this downgrade does not break anything (but fixes plotnine), I guess that it is justified.

Reply:
Also, only two packages depend on `mizani`: `bsuite` and `plotnine` which latest versions both work fine with `mizani`==0.9.3

Reply:
The downgrade is reasonable, but the commit message is backwards.

Reply:
Indeed, fixed.